Feat: Button

vivekmehto opened the pull request:
https://user-images.githubusercontent.com/76930847/152111510-a72f7b66-3afc-4a74-8030-6d29842ab9b5.mov

vivekmehto committed:
ADD button.css

vivekmehto committed:
Fix: float btn position

vivekmehto committed:
Update button.css

ttahm3d commented on components/button/button.css:
```diff
@@ -0,0 +1,100 @@
+.btn {
+  margin: 0.5rem;
```
Dont give margin and padding both inside component. Try to handle margin between elements using flex gap to the outer container

ttahm3d commented on components/button/button.css:
```diff
@@ -55,7 +55,7 @@
 .btn-float {
   position: absolute;
```
as far as I think this should be position fixed. absolute will move out of viewport on scroll

ttahm3d reviewed (commented):
float btns should be fixed to viewport. Other than that not many changes

ImrozNezami commented:
You can try to do changes mentioned above apart from that all looks good.

ImrozNezami commented on components/button/button.css:
```diff
@@ -0,0 +1,102 @@
+}
+
+.btn-float {
+  position: absolute;
```
You can try to changes mentioned here apart from that all looks good.

vivekmehto committed:
Fix: fab position

changed floating action button position to fixed and removed margin from btn class.
